diff --git a/node_modules/react-native-cashfree-pg-sdk/android/build.gradle b/node_modules/react-native-cashfree-pg-sdk/android/build.gradle
index 826f4c4..1f8d256 100644
--- a/node_modules/react-native-cashfree-pg-sdk/android/build.gradle
+++ b/node_modules/react-native-cashfree-pg-sdk/android/build.gradle
@@ -17,7 +17,7 @@ android {
   namespace "com.reactnativecashfreepgsdk"
     defaultConfig {
       minSdkVersion 21
-      compileSdk 35
+      compileSdk 36
     }
 
     buildTypes {
